python/xorbits/_mars/dataframe/datasource/read_parquet.py

```diff
@@ -651,6 +668,15 @@ def read_parquet(
     single_path = path[0] if isinstance(path, list) else path
     fs = get_fs(single_path, storage_options)
     is_partitioned = False
+    if use_arrow_dtype is None:
+        use_arrow_dtype = options.dataframe.use_arrow_dtype
+    if use_arrow_dtype and engine_type != "pyarrow":
+        raise ValueError(
+            f"`engine` should be pyarrow instead of {engine_type}"
+            f" when use arrow dtype as extension dtype"
```

Comment:
In pandas 2.0, specifying `dtype_backend` along with `engine='fastparquet'` raises the following error: 
`The 'dtype_backend' argument is not supported for the fastparquet engine`

The error message could be simplified accordingly:

```suggestion
            f"The 'use_arrow_dtype' argument is not supported for the {engine_type} engine"
```